density.c

```diff
@@ -126,66 +123,7 @@ void density(void)
 
     walltime_measure("/Misc");
 
-    DENSITY_GET_PRIV(tw)->Left = (double *) mymalloc("DENSITY_GET_PRIV(tw)->Left", NumPart * sizeof(double));
-    DENSITY_GET_PRIV(tw)->Right = (double *) mymalloc("DENSITY_GET_PRIV(tw)->Right", NumPart * sizeof(double));
-
-    DENSITY_GET_PRIV(tw)->NIteration = 0;
-
-    /* this has to be done before treewalk so that
-     * all particles are ran for the first loop.
-     * The iteration will gradually turn DensityIterationDone on more particles.
-     * */
-    #pragma omp parallel for
-    for(i = 0; i < NumActiveParticle; i++)
-    {
-        const int p_i = ActiveParticle[i];
-        P[p_i].DensityIterationDone = 0;
-        DENSITY_GET_PRIV(tw)->Left[p_i] = 0;
-        DENSITY_GET_PRIV(tw)->Right[p_i] = 0;
-
-    }
-
-    /* allocate buffers to arrange communication */
-
-    walltime_measure("/SPH/Density/Init");
```

Comment:
Can you change the names of the walltime indicators in sml to read something else? eg, "SPH/SML/Init"